src/bitwarden_exporter/settings.py

```
@@ -104,37 +105,41 @@ def get_bitwarden_settings_based_on_args() -> BitwardenExportSettings:
         default=False,
     )
 
-    with open(
-        os.path.join(os.path.dirname(os.path.realpath(__file__)), "resources", "version.txt"),
-        encoding="utf-8",
-    ) as f:
-        cli_version = f.read()
+    try:
+        uv_version = version(constants.APPLICATION_PACKAGE_NAME)
+    except PackageNotFoundError:
+        raise SystemExit(f"Package {constants.APPLICATION_PACKAGE_NAME} not found")
```

Ruff (0.14.1)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)